### PR TITLE
Implemented async/await

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**

### CODE REVIEW

File: src/passport/strategies/user/strategy-facebook.js [hunk 1]
```diff
@@ -16,104 +16,102 @@ const tracer = require('../../../utils/ddtracer').tracer
...
-                            return cb(err, null)
-                        })
-                }
-            })
-            .catch((err) => {
-                Raven.captureException(err)
-                return cb(null, false, {message: err.message})
-            })
-    }
-    else {
```
Comment (championswimmer) [inline src/passport/strategies/user/strategy-facebook.js]:
```
if {
...
} else {
...
}
```

please use this indentation scheme

Comment (VibhorCodecianGupta) [reply src/passport/strategies/user/strategy-facebook.js]: Done! @championswimmer

**Comment by championswimmer** [comment]
login doesn't work
I get "Authentical failed" error.

**Comment by VibhorCodecianGupta** [comment]
@championswimmer which strategy? Local?

**Comment by championswimmer** [comment]
no social ones.

**Comment by championswimmer** [comment]
.spread() is array not object. you got that wrong

**Comment by VibhorCodecianGupta** [comment]
Just pushed a commit, I think this will solve the issue.

**Comment by VibhorCodecianGupta** [comment]
@championswimmer was there an issue with this PR?

**Comment by VibhorCodecianGupta** [comment]
@championswimmer what was the issue with this PR?

**Pull request closed without merging**